frontend/src/components/editor/cell/code/cell-editor.tsx

```diff
@@ -410,3 +434,29 @@ const CellCodeMirrorEditor = React.forwardRef(
 CellCodeMirrorEditor.displayName = "CellCodeMirrorEditor";
 
 export const CellEditor = memo(CellEditorInternal);
+
+export const getWordUnderCursor = (state: EditorState) => {
```

Comment:
i think we can move all this logic to another file `core/codemirror/go-to-definition.ts`. 
we should be able to just expose one function `goToDefinition(view: EditorView)`

Reply:
Perfect